packages/cli/bin/merkur.mjs

```diff
@@ -138,4 +141,22 @@ program
     await custom({ args, commandArgs: cmd.args, command: COMMAND_NAME.CUSTOM });
   });
 
+// Load user-defined commands
+let userDefinedCommands = [];
+for (const { command, dir } of userDefinedCommandsPaths) {
+  const programCustom = new Command();
+  const commandModule = await import(path.join(dir, command));
```

Comment:
I would add try/catch block for import and default calling function so merkur CLI work if one of custom command fails.

Reply:
Pridan try-catch